### PR TITLE
Pull request for Library App project.

### PR DESCRIPTION
**Pull request by FdI96**
Kindly request a code review.

This app has the following features: 
Has a form to add a book with its name, author, pages amount, and if It has been read
Shows books saved.
Has a button to remove a book.
Has a button to change the read status.

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Federico Lopez Cechini

YOU HAVE DONE A GOOD JOB ON THIS PROJECT:+1:

## To Highlight :tada:


1. Your PR has a good title and description.
2. You have descriptive git commit messages.
3. You have mastered the Git flow


 There are some changes that you need to make in order to get your project approved

- [x] It has a constructor for creating book object

      ex. 
    
      function Book(name, author, pages, read) {
        this.name = name;
        this.author = author;
        this.pages = pages;
        this.read = read;
      }

In your case the [arrow function can not be used as constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), because arrow functions cannot be called with the new keyword


- [x] Remove all the```console.log```s 

- [x] There should be ```addBookToLibrary function```, which will take the user's input and store the new Book objects into an array [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png)

- [x] Create a function that loops through the array and displays each book [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/books_render_loop.png)

<hr>

## OPTIONAL

- [ ] The form used to create a book should have validations, you should not be able to create an empty book

- [ ] It has data persistence by saving, retrieving, and updating the library books using [the local storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API)

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const bookObjectMaker = (name, author, pages, read) => {
+  const book = {
+    name,
+    author,
+    pages,
+    read,
+  };
+
+  return book;
+};
```
Comment (MkrtichSargsyan) [inline index.js]: an arrow function can not be used as a constructor

File: index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  itemPages.innerHTML = book.pages;
+  list.appendChild(itemAuthor);
+  list.appendChild(itemPages);
+  list.appendChild(itemRead);
+  bigTag.appendChild(list);
+  return bigTag;
+}
+
+function showBooksList(book) {
+  console.log(book);
```
Comment (MkrtichSargsyan) [inline index.js]: remove the ```console.log```

File: index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  const elementAuthor = document.getElementById('author');
+  valuesArr.push(elementAuthor.value);
+  const elementPages = document.getElementById('pages');
+  valuesArr.push(elementPages.value);
+  const elementRead = document.getElementById('read');
+  valuesArr.push(elementRead.checked);
+  const book = bookObjectMaker(...valuesArr);
+  bookArr.push(book);
+  showBooksList(book);
+  console.log(bookArr);
```
Comment (MkrtichSargsyan) [inline index.js]: remove the ```console.log```

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- It has a constructor for creating book object :+1: 
- There is a addBookToLibrary function :100: 

To improve :factory_worker: [REQUIRED]:

- [ ] Create a function that loops through the array and displays each book [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/books_render_loop.png)

Nice to have:

- [ ]  It has data persistence by saving, retrieving, and updating the library books using the [local storage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API)

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Comment by eliasecasta** [comment]
## Status:  Approved 🟢

Hello @FdI96,

Thank you for applying the required changes from the past TSE, good luck with your next projects!

**Pull request merged**